README.md

```diff
@@ -55,6 +55,8 @@ these priorities may not be accepted.
 Please familiarize yourself with the [Contribution
 Guide](https://flynn.io/docs/contributing) before contributing.
 
+You can install flynn development environment and tools by following the [Development Guide](https://flynn.io/docs/development).
```

Comment:
Oh I did notice that in the first place but I seem to forgot to press `cmd+s` on sublime text.